pkg/lib/bundle/generate.go

```diff
@@ -286,3 +364,52 @@ func WriteFile(fileName, directory string, content []byte) error {
 	}
 	return nil
 }
+
+// copy the contents of a flat manifest dir into an output dir
+func copyManifestDir(from, to string, overwrite bool) error {
+	fromFiles, err := ioutil.ReadDir(from)
+	if err != nil {
+		return err
+	}
+
+	if _, err := os.Stat(to); os.IsNotExist(err) {
+		os.MkdirAll(to, os.ModePerm)
+	}
+
+	for _, fromFile := range fromFiles {
+		// ignore nested directories
```

Comment:
this might cause us problems if we want to support helm charts, kustomize, etc

Reply:
That's a fair point, I'll update this to copy recursively